src/context.js

```diff
@@ -188,11 +267,8 @@ function EvolvContext(store) {
        ' and "contaminations" is deprecated. Please use "experiments.confirmations" and "experiments.contaminations" instead.');
     }
 
-    const valueFromRemote = objects.getValueForKey(key, remoteContext);
-
-    return objects.hasKey(key, remoteContext)
-      ? valueFromRemote
-      : objects.getValueForKey(key, localContext);
+    return objects.getValueForKey(key,
+      objects.hasKey(key, remoteContext)? remoteContext : localContext);
```

Comment:
These lines were deliberately changed to address a bug with falsy values. Can you make sure this doesn't cause a regression?

See https://github.com/evolv-ai/javascript-sdk/pull/128/files#diff-f00be3776778d4a72e4f0720b43138c8a7f09e508f31a1987b8952410b315518

Reply:
The only boolean expression here is the result of hasKey, which has a boolean response type?

Reply:
Yes, this should be good then.